frappe/__init__.py

```diff
@@ -91,14 +90,14 @@ def _(msg, lang=None, context=None):
 
 def as_unicode(text, encoding='utf-8'):
 	'''Convert to unicode if required'''
-	if isinstance(text, text_type):
+	if isinstance(text, str):
 		return text
 	elif text==None:
 		return ''
-	elif isinstance(text, binary_type):
-		return text_type(text, encoding)
+	elif isinstance(text, bytes):
+		return str(text, encoding)
 	else:
-		return text_type(text)
+		return str(text)
```

Comment:
Can this entire function be let go? Unicode was removed as a datatype in Py3.

Reply:
We can do that, will take care!

Reply:
as_unicode is required when we are working with `bytes` and wanted to decode it with specific encoding type. So, leaving that method as is.